apps/web/public/static/locales/en/common.json

```diff
@@ -2322,7 +2322,13 @@
   "success_entry_created": "Successfully created a new entry",
   "booking_redirect_email_subject": "Booking redirect notification",
   "booking_redirect_email_title": "Booking Redirect Notification",
-  "booking_redirect_email_description": "You have received a booking redirection from {{toName}} so their profile links will be redirect to yours for the time interval: ",
+  "booking_redirect_email_description": "You have received a booking redirection from {{toName}} so their profile links will be redirect to yours for the time interval: \"{{dates}}\"",
+  "booking_redirect_edit_email_subject": "Booking redirect edit notification",
+  "booking_redirect_edit_email_title": "Booking Redirect Edit Notification",
+  "booking_redirect_edit_email_description": "Your booking redirection from {{toName}} for interval \"{{oldDates}}\" have been edited. New time interval for the redirection is: \"{{dates}}\"",
+  "booking_redirect_cancel_email_subject": "Booking redirect cancel notification",
+  "booking_redirect_cancel_email_title": "Booking Redirect Cancel Notification",
+  "booking_redirect_cancel_email_description": "Your booking redirection from {{toName}}, for time interval \"{{dates}}\" have been canceled.",
```

Comment:
Hey, can we send a cancellation notification to the team member if an out-of-office entry is deleted? If not, then send it

Reply:
@anikdhabal No previously on canceling an event email was not send. I thought of adding it, but didn't do it. But now done. 👍